crates/uv-python/download-metadata.json

```diff
@@ -47,503 +47,7 @@
     "sha256": "2d61c25acb85f71385e358ca7f658b9d2ae5971161985c32b182edd2d75da4a6",
     "variant": null
   },
-  "cpython-3.14.0a7-linux-armv7-gnueabi": {
-    "name": "cpython",
-    "arch": {
-      "family": "armv7",
-      "variant": null
-    },
-    "os": "linux",
```

Comment:
Unfortunate that entire versions of cpython go away...

Reply:
We should highlight this in the release, saying that these prereleases are temporarily removed due to an unrelated, non-alpha-related miscompilation.